SourceX/miniwin/misc.cpp

```diff
@@ -178,13 +178,13 @@ HWND CreateWindowExA(
 	DvlIntSetting("grab input", &grabInput);
 
 #ifdef USE_SDL1
-	int flags = SDL_SWSURFACE | SDL_DOUBLEBUF | SDL_HWPALETTE;
+	int flags = SDL_SWSURFACE /*| SDL_DOUBLEBUF */| SDL_HWPALETTE;

```

Comment:
Why disable this?
We use SDL_Flip when rendering with SDL 1 https://www.libsdl.org/release/SDL-1.2.15/docs/html/sdlflip.html

Reply:
Looks like it was added and disabled by @arczi84 during the big merge. Probably just need to be tested and then enabled.

Reply:
At least on Amiga it's doesn't work, so #ifdef for Amiga should be here.

Reply:
Possibly this should be moved to the SDL layer since it's implementation specific

Reply:
Handled this in Amiga.